Library Genesis.js

```diff
@@ -0,0 +1,245 @@
+{
+	"translatorID": "11614156-f421-4e89-8ce0-a5e69ce3ebed",
+	"label": "Library Genesis",
+	"creator": "Reverend Wicks Cherrycoke",
+	"target": "libgen.io|lib.gen.rus.ec",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsi",
+	"lastUpdated": "2016-04-22 22:11:28"
+}
+
+var MIME_TYPES = {
+  'pdf': 'application/pdf',
+  'epub': 'application/epub+zip',
+  'mobi': 'application/x-mobipocket-ebook',
+  'djvu': 'image/vnd.djvu'
+};
+var MD5_REGEX = /md5=([0-9a-fA-F]+)/;
+
+
+function detectWeb(doc, url) {
+  if (url.indexOf('book/index.php') != -1) {
+	return 'book';
+  }
+}
+
+function doWeb(doc, url) {
+  var md5Hash = MD5_REGEX.exec(url)[1];
+
+  // To save some work, we use the provided bibtex file to retrieve the
+  // metadata and use Zotero's built-in bibtex importer
+  var bibtexUrl = "http://libgen.io/book/bibtex.php?md5=" + md5Hash;
+  ZU.processDocuments(bibtexUrl, function(bibtexDoc) {
+	var bibtexStr = bibtexDoc.getElementsByTagName("textarea")[0].value
+	var translator = Zotero.loadTranslator('import');
+	translator.setTranslator('9cb70025-a888-4a29-a210-93ec52da40d4');
+	translator.setString(bibtexStr);
+
+	translator.setHandler('itemDone', function(obj, item) {
+	  // The bibtex messes up multiple authors, so we set them ourselves
+	  var authorStr = ZU.xpathText(doc, "//tbody/tr[3]/td[2]/b");
```

Comment:
why not parse this from the bibtex? That seems more robust and also means you don't handle two documents at the same time. I'm worried who well that'll work e.g. in the bookmarklet. Also makes it harder to work with multiples later on.